Core/include/Acts/Propagator/ConstrainedStep.hpp

```diff
@@ -151,6 +168,8 @@ class ConstrainedStep {
 
   /// the step size tuple
   std::array<Scalar, 4> m_values = {kNotSet, kNotSet, kNotSet, kNotSet};
```

Comment:
This could become size 3 then no? Also, why split up the storage of the accuracy value? I get the argument about the setter but the storage is not clear to me.

Reply:
right that should not be 4 anymore

about the splitting: the array is addressed via the enum and `update` is also using the type to perform its job. I wanted to exclude `accuracy` from `update` and the best way was to remove it from the enum